middlewares/auth.js

```diff
@@ -0,0 +1,23 @@
+const jwt = require("jsonwebtoken");
+
+const loginRequired = async (req, res, next) => {
+  try {
+    const token = req.headers.authorization;
+    if (!token) {
+      const error = new Error("YOU NEEDS TOKEN!!");
+      error.statusCode = 400;
+      throw error;
+    }
+    const SECRET_KEY = process.env.SECRET_KEY;
+    const decode = jwt.verify(token, SECRET_KEY);
+
+    req.user = decode.id;
+    next();
```

Comment:
return 키워드를 통해 해당 함수가 종료됨을 표현할 수 있습니다.
추가로 next() 함수 이후의 예기치 못한 코드 실행을 방지할 수 있습니다.
```suggestion
    return next();
```